honeybee_radiance_command/rfluxmtx.py

```diff
@@ -0,0 +1,139 @@
+"""rfluxmtx command"""
+
+from ._command import Command
+import honeybee_radiance_command._exception as exceptions
+import honeybee_radiance_command._typing as typing
+from .options.rfluxmtx import RfluxmtxOptions, RfluxmtxControlParameters
+
+
+# octree to octree and so on
+class Rfluxmtx(Command):
+    """
+    Rfluxmtx command.
+
+    Rfluxmtx computes the flux transfer matrices for a RADIANCE scene.
+
+    """
```

Comment:
I would add more documentation here and include args, etc. See rcontrib as an example:

https://github.com/ladybug-tools/honeybee-radiance-command/blob/1a3b7d3247ac3f5fd3ddb494d4d6dfb3ee0c4a73/honeybee_radiance_command/rcontrib.py#L7-L31